tests/executor.cpp

```diff
@@ -10,7 +10,7 @@ using namespace mips_emulator;
 using Func = Instruction::Func;
 using IOp = Instruction::ITypeOpcode;
 
-TEMPLATE_TEST_CASE("add", "[Executor]", RegisterFile32, RegisterFile64) {
+TEMPLATE_TEST_CASE("add", "[Executor]", RegisterFile32) {
```

Comment:
You can change this from a TEMPLATE_TEST_CASE into a normal TEST_CASE because it only contains one type, see for example the Instruction tests in tests/instruction.cpp